config/config.go

```diff
@@ -71,6 +71,10 @@ func normalizeDir(dir string) (string, error) {
 		return dir, nil
 	}
 
+	if xdg := os.Getenv("XDG_CONFIG_HOME"); xdg != "" {
```

Comment:
Can we make sure that we use `~/.jump` if present, so we don't break the existing installs (or force them to move to xdg)? Have you tested that?

Reply:
I think it's better to keep ~/.jump if present. I try it.

Reply:
Thanks!